packages/components/src/components/form/BottomText.tsx

```diff
@@ -36,13 +35,15 @@ interface BottomTextProps {
     inputState?: InputState;
     isDisabled?: boolean;
     icon?: IconName;
+    renderIcon?: ReactNode;
     children: ReactNode;
 }
 
 export const BottomText = ({
     inputState,
     isDisabled,
     icon = 'warningCircle',
+    renderIcon,
```

Comment:
I know I was suggesting `renderIcon`, but I changed my mind a bit :D 

I suggest to delete icon and instead of `renderIcon` rename it to `iconComponent`. We would accept only react element here instead of string for icon. 

It's problematic to have two same props for the same purpose, you have to check how it's implemented to understand what has the priority if you use it this way?:


```tsx
<BottomText
  icon="SHOW"
  renderIcon={<Icon name="HIDE" />}  
/>

```

WDYT?


So it could be just:

```tsx
<BottomText
  iconComponent={<Icon name="HIDE" />}  
/>

```